components/chassis.py

```diff
@@ -150,6 +150,9 @@ def _get_heading(self) -> Rotation2d:
         """Get the current heading of the robot from the IMU, anticlockwise positive."""
         return self.imu.getRotation2d()
 
+    def reset_heading(self) -> None:
+        self.imu.reset()
```

Comment:
I think this will have bad consequences for the odometry. Can you reset the odometry instead?